airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -480,6 +480,15 @@ def __init__(
         self.waiter_delay = waiter_delay
         self.waiter_max_attempts = waiter_max_attempts
 
+        if self._aws_logs_enabled() and not self.wait_for_completion:
+            self.log.warning(
+                "Trying to get logs without waiting for the task to complete is undefined behavior."
+            )
+
+    @staticmethod
+    def _get_ecs_task_id(task_arn: str) -> str:
+        return task_arn.split("/")[-1]
```

Comment:
doing this because I thought that remembering to update the ecs_task_id every time the arn gets changed was a bit brittle. This method makes the dependency arn->task_id explicit.
Yes it means we're going to recompute it each time we need it, but I think it's negligible.